app/Http/Services/Tabbar/TabbarService.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace App\Http\Services\Tabbar;
+
+use Illuminate\Http\Request;
+use App\Http\PathInfo;
+
+/**
+ * TabbarService
+ *
+ * タブバーの加工を行う
+ *
+ */
+class TabbarService {
+
+    /**
+     * Create a new docs controller instance.
+     *
+     * @return void
+     */
+    public function __construct() {
+        
+    }
+
+    /**
+     * タブの情報を取得する
+     * 
+     * @param Request リクエストパラメータ
+     * @return array 各tabで表示するためのURL
+     * 
+     */
+    public function getTabbarInfo($request) {
+        // リクエストパラメータからタブの情報を取得する
+        $tabbar = $request->only([
+            'home',
+            'contents',
+            'docs',
+            'dev'
+        ]);
+        // PathInfoのインスタンス取得
+        $pathInfo = new PathInfo();
+        return [
+            'HOME' => $this->_getHomeTabInfo($tabbar['home'], $pathInfo->getHomeTabUrlInfo()),
+            'CONTENTS' => $this->_getContentsTabInfo($tabbar['contents'], $pathInfo->getContentsTabUrlInfo()),
+            'DOCS' => $this->_getDocsTabInfo($tabbar['docs'], $pathInfo->getDocsTabUrlInfo()),
+            'DEV' => $this->_getDevTabInfo($tabbar['dev'], $pathInfo->getDevTabUrlInfo())
+        ];
+    }
+
+    /**
+     * Homeタブの情報を取得する
+     * 
+     * @param string $homeParam リクエストパラメータのhomeの値
+     * @param array $homeTabUrlInfo Homeタブ内に表示するページのURL一覧
+     * @return string Homeタブに表示するページのURL
+     * 
+     */
+    private function _getHomeTabInfo($homeParam, $homeTabUrlInfo) {
+
+        // パラメータからHomeタブのlistのキー値を作成する
+        $listkey = strtoupper($homeParam);
+
+        // homeパラメータ値がhomeタブに存在するurlと一致しない場合
+        if (!array_key_exists($listkey, $homeTabUrlInfo)) {
+            // home_topを表示させる
+            return $homeTabUrlInfo['TOP'];
+        } else {
```

Comment:
__[phpmd]__
* The method _getHomeTabInfo uses an else expression. Else is never necessary and you can simplify the code to work without else.